packages/@aws-cdk/aws-apigateway/test/test.domains.ts

```diff
@@ -65,6 +65,41 @@ export = {
     test.done();
   },
 
+  'accepts different security policies'(test: Test) {
```

Comment:
Can you add a 3rd verification, that when left unspecified, the value is absent in the CF template. Use [`ABSENT`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/assert/README.md#check-existence-of-a-resource) to do this.